source/common/stream_info/stream_info_impl.h

```diff
@@ -259,7 +371,8 @@ struct StreamInfoImpl : public StreamInfo {
 
   void dumpState(std::ostream& os, int indent_level = 0) const {
     const char* spaces = spacesForLevel(indent_level);
-    os << spaces << "StreamInfoImpl " << this << DUMP_OPTIONAL_MEMBER(upstream_connection_id_)
+    // TODO(alyssawilk) dump upstream info.
```

Comment:
I should do this one before I commit.
quick question before I do though, do you see any value in having the abstract API here, rather than just making upstream info an explicit struct and exposing the member variables?  I can go either way but don't want to plumb until we decode.